### PR TITLE
[CDAP-14850] Update react-calendar to latest version to fix bug

### PR DESCRIPTION
**Pull request by yukiej**
JIRA: https://issues.cask.co/browse/CDAP-14850
BUILD: https://builds.cask.co/browse/CDAP-UDUT360

Fixes bug for setting year/month of custom date range. Updated react-calendar from 2.15.0 to 2.19.0.

### CODE REVIEW

**Comment by itsanudeep** [comment]
LGTM 👍

**Review by elfenheart — APPROVED** [review]

**Comment by yukiej** [comment]
Currently working on adding an e2e test for the reports to add to this PR.

File: cdap-ui/cypress/fixtures/fll_airport_pipeline.json [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+                    "top": "239.5px"
+                }
+            }
+        ],
+        "schedule": "0 * * * *",
+        "engine": "mapreduce",
+        "numOfRecordsPreview": 100,
+        "maxConcurrentRuns": 1
+    }
+}
```
Comment (elfenheart) [inline cdap-ui/cypress/fixtures/fll_airport_pipeline.json]: nit: newline at EOF

File: cdap-ui/cypress/integration/reports.spec.ts [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+*/
+
+import * as Helpers from '../helpers';
+
+let headers = {};
+
+describe('Generating reports for pipelines', () => {
```
Comment (elfenheart) [inline cdap-ui/cypress/integration/reports.spec.ts]: this doesn't seem to contain any test. Delete the file?

Comment (yukiej) [reply cdap-ui/cypress/integration/reports.spec.ts]: Deleted, thanks!

**Review by elfenheart — APPROVED** [review]
👍 LGTM

**Review by ajainarayanan — APPROVED** [review]
Had a minor comment about the height of existing FLL page and calendar getting cropped by it. Otherwise 👍 LGTM

**Comment by yukiej** [comment]
RAT and Checkstyle: https://builds.cask.co/browse/CDAP-DRC9370

**Comment by yukiej** [comment]
e2e Tests: https://builds.cask.co/browse/IT-UIE2EMINI1-2

**Comment by ajainarayanan** [comment]
[Build #1](https://builds.cask.co/build/result/gotoBuildResult.action?buildKey=IT-UIE2EMINI2&buildNumber=13&returnUrl=%2Fbrowse%2FIT-UIE2EMINI2-13) [Build#2](https://builds.cask.co/build/result/gotoBuildResult.action?buildKey=IT-UIE2EMINI2&buildNumber=12&returnUrl=%2Fbrowse%2FIT-UIE2EMINI2-12) Two consecutive builds passed. Merging. 👏 👏 🎆 👩‍🚒 Great work @yukiej !

**Pull request merged**